bin/anthology/index.py

```diff
@@ -230,7 +230,7 @@ def create_bibkey(self, paper):
         self.bibkeys.add(bibkey)
         return bibkey
 
-    def register(self, paper):
+    def register(self, paper, dummy=False):
```

Comment:
Can you add a comment here so people in a decade (or me next week) know what the dummy argument is for?

Reply:
Will do. The cleaner alternative might be to factor out the different parts (that are influenced by the `dummy` flag) into separate functions, but maybe that's overkill for now ...